ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -117,17 +124,99 @@ type opennebula_tm_mad_conf = {
     "ln_target" : string = "NONE"
     "clone_target" : string = "SYSTEM"
     "shared" : boolean = true
+    "ds_migrate" ? boolean
+} = dict();
+
+@documentation{
+The  configuration for each driver is defined in DS_MAD_CONF.
+These values are used when creating a new datastore and should not be modified
+since they define the datastore behavior.
+name: name of the transfer driver, listed in the -d option of the DS_MAD section.
+required_attrs: comma separated list of required attributes in the DS template.
+persistent_only: specifies whether the datastore can only manage persistent images.
+}
+type opennebula_ds_mad_conf = {
+    "name" : string = "dummy"
+    "required_attrs" : string = ""
```

Comment:
Is this a required field? If yes, is an empty string a valid default? (If no, also optional in schema?)